### DISCUSSION TITLE
Store Docker Images in AWS ECR Public

### DISCUSSION THREAD

**Discussion by toneill818** [apache/pulsar#18782] [General]
**Is your enhancement request related to a problem? Please describe.**
DockerHub has aggressive rate limits now and impacts the usage when running Apache Pulsar on Kubernetes, if the rate limit is reached. 

**Describe the solution you'd like**
AWS created the public ECR with not rate limiting. The solution could be to store images in Dockerhub and AWS ECR and possibly Google's GCR. This will allow users to choose which service they pull from and avoid the issue with Docker Hub's rate limiting. 

**Describe alternatives you've considered**
We can pull the image and push it to a private repository and use that to pull from (what I am currently doing) to mirror the Docker Hub repo, but it would be nice to not to have to manually update it.

**Comment by hpvd** [comment]
To make it available from as many places as possible, is also a way to make pulsar even more visible  (Marketing) and grow the community and adoption :-)
